tfx/examples/imdb/README.md

```diff
@@ -0,0 +1,46 @@
+# IMDB Sentiment Analaysis Example
+
+* The data is downloaded from tensorflow_dataset.
+* The example dataset imdb_small.csv that comes within /data is the first 100 entries of the original dataset.
+* To fetch the entire dataset, please use the snippet below 
+* And please adjust the corresponding hyperparameters to account for the larger dataset.
```

Comment:
Nit: Not sure if these should all be bullets or just a paragraph. Also, if they should be bullets, we should condense that last two into a single one.

In general, this README looks different from the other example READMEs. That said, others have signed off, and I'm satisfied leaving as is if they are.